src/Freya.Core/Operators.fs

```diff
@@ -20,11 +20,16 @@
 /// Custom operators for composing <see cref="Freya{T}" /> computations.
 module Freya.Core.Operators
 
+(* Compositional
+
+   Operators for the composition of Freya<'T> functions in
+   various forms. *)
+
 let inline (>>=) m f =
-    Freya.bind m f
+    Freya.bind f m
```

Comment:
I flipped the operators on `Freya.bind` so that you could start a pipeline, e.g.

``` fsharp
m |> Freya.bind (fun b -> Lens.setState someLens (1::b))
  |> Freya...
```